### PR TITLE
To add Qradar Analytics rules Resource module

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

To add Qradar Analytics rules Resource module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

qradar_analytics_rules

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[tool.black]
+line-length = 79
```
Comment (KB-perByte) [inline pyproject.toml:2]:
```suggestion
line-length = 100
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      - id: debug-statements
+      - id: end-of-file-fixer
+      - id: no-commit-to-branch
+        args: [--branch, main]
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args: [-l, "79"]
```
Comment (KB-perByte) [inline .pre-commit-config.yaml:17]:
```suggestion
        args: [-l, "100"]
```

**Review by KB-perByte — APPROVED** [review]

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    uses: ansible-network/github_actions/.github/workflows/sanity.yml@main
+  unit-galaxy:
+    uses: ansible-network/github_actions/.github/workflows/unit_galaxy.yml@main
+  unit-source:
+    uses: ansible-network/github_actions/.github/workflows/unit_source.yml@main
+    with:
+      collection_pre_install: >-
+        git+https://github.com/ansible-collections/ansible.utils.git
+        git+https://github.com/ansible-collections/ansible.netcommon.git
+  all_green:
```
Comment (webknjaz) [inline .github/workflows/test.yml:26]: @justjais don't forget to list this job in branch protection.

cc @gundalow

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  changelog:
+    uses: ansible-network/github_actions/.github/workflows/changelog.yml@main
```
Comment (webknjaz) [inline .github/workflows/test.yml:15]: @justjais FYI https://github.com/ansible-community/ansible-test-gh-action is a thing that is now integrated into the default collection template @ https://github.com/ansible-collections/collection_template and will eventually be a part of ansible-core's suggested skeleton too.

Comment (justjais) [reply .github/workflows/test.yml:15]: @webknjaz thanks so much for the information.

**Pull request merged**